pages/about.js

```diff
@@ -0,0 +1,300 @@
+// @flow
+import Layout from "../components/layout/layout";
+import {ContentSection, ContentSectionPane} from "../components/content-section/content-section";
+import React from "react";
+
+import {
+    PageHeader,
+    PageHeaderLeft,
+    PageHeaderRight
+} from "../components/page-header/page-header";
+
+const contributorsData = [
```

Comment:
Created [`fetch-github-organization`](https://github.com/rgbkrk/fetch-github-organization) for this -- we can use it in `getInitialProps` since it returns a promise. Needs error handling, works as a first pass for now. 😄 

Admittedly though, because of rate limiting it might make more sense to fetch the user list before `npm run start` and rely on that data being available as the cached version. :wink: